src/mumble/Log.ui

```diff
@@ -193,7 +198,27 @@
         </property>
        </widget>
       </item>
-     </layout>
+	  <item>
+        <widget class="QCheckBox" name="qcbMessageThreshold">
+         <property name="toolTip">
+          <string>If checked, notifications will not be played when more than 20 people are active on the server.</string>
+         </property>
+         <property name="text">
+          <string>Don't play notifications when there are more than the following number of users on the server.</string>
+         </property>
+		</widget>
+       </item>
+       <item>
+	    <widget class="QSpinBox" name="qsbMessageThresholdUsers">
+	     <property name="toolTip">
+		  <string>Number of users that will trigger thresholding functionality.</string>
+		  </property>
+		  <property name="frame">
+		   <bool>true</bool>
+		  </property>
+		 </widget>
+       </item>
+	 </layout>
```

Comment:
Also, I tried to implement this like the following. 
```
	  <item>
        <widget class="QCheckBox" name="qcbMessageThreshold">
         <property name="toolTip">
          <string>If checked, notifications will not be played when more than 20 people are active on the server.</string>
         </property>
         <property name="text">
          <string>Don't play notifications when there are more than 
<widget class="QSpinBox" name="qsbMessageThresholdUsers">
	     <property name="toolTip">
		  <string>Number of users that will trigger thresholding functionality.</string>
		  </property>
		  <property name="frame">
		   <bool>true</bool>
		  </property>
		 </widget>
 number of users on the server.</string>
         </property>
		</widget>
       </item>
	 </layout>
```




It built/compiled fine, but crashed immediately when I opened the settings page. It obviously didn't like trying to put a spinbox widget in the middle of a string. I think I knew what to do next, but I didn't want to change the layout from "verticalLayout" to "gridLayout" (like the other settings box immediately below this one) prematurely.

Reply:
I can't help you with manually editing the UI files. Never did that and I strongly recommend against doing so if the changes are not trivial :point_up: